spec/spec-helper.js

```diff
@@ -49,10 +49,10 @@ Set.prototype.jasmineToString = function() {
 
 Set.prototype.isEqual = function(other) {
   if (other instanceof Set) {
-    let next;
     if (this.size !== other.size) { return false; }
     const values = this.values();
-    while (!(next = values.next()).done) {
+    let next = values.next();
+    while (!next.done) {
       if (!other.has(next.value)) { return false; }
```

Comment:
This actually changes the while loop - while loops evaluate their condition every iteration
```suggestion
    let next = values.next();
    while (!next.done) {
      if (!other.has(next.value)) { return false; }
      next = values.next();
```

Reply:
Thanks, unfortunately it's hard to not mistake some of these. Committed your suggestion.